CMakeLists.txt

```diff
@@ -103,6 +103,7 @@ if(MX_BUILD_TESTS)
     target_link_libraries(MxTest ${PROJECT_NAME})
     target_link_libraries(MxTest ${CMAKE_THREAD_LIBS_INIT})
     set_property(TARGET MxTest PROPERTY CXX_STANDARD 14)
+    set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT MxTest)
```

Comment:
Makes MxTest the default target in MSVC.

Reply:
👍